src/shared/components/cohort/Icon.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import DefaultTooltip from "../DefaultTooltip";
+
+export interface IIconProps
+{
+    text: string;
+    tooltip?: JSX.Element;
+    stroke?: string;
+    circleFill?: string;
+    fontSize?: number;
+    textFill?: string;
+}
+
+/**
+ * @author Selcuk Onur Sumer
+ */
+export default class Icon extends React.Component<IIconProps, {}>
+{
+    public static defaultProps = {
+        stroke: "#55C",
+        circleFill: "none",
+        fontSize: 7,
+        textFill: "#66C"
+    };
+
+    public render()
+    {
+        let content = (
+            <svg width="12" height="12">
+                <circle r="5" cx="6" cy="6" stroke={this.props.stroke} fill={this.props.circleFill}/>
+                <text x="3" y="8.5" fontSize={this.props.fontSize} fill={this.props.textFill}>
+                    {this.props.text}
```

Comment:
It's going to look crazy with more than one letter, right? Maybe you can explicitly only put in the first letter of the string?